tests/phpunit/tests/mail/test-phpmailer-translations.php

```diff
@@ -1,18 +1,64 @@
 <?php
 
 class Test_PHPMailer_Translations extends WP_UnitTestCase {
+
+	private $original_locale;
+	private $mo_file;
+
+	public function set_up(): void {
+		parent::set_up();
+
+		$this->original_locale = get_locale();
+		$this->mo_file         = DIR_TESTDATA . '/languages/de_DE.mo';
+
+		load_textdomain( 'default', $this->mo_file );
+		switch_to_locale( 'de_DE' );
+	}
+
+	public function tear_down(): void {
+		switch_to_locale( $this->original_locale );
+		unload_textdomain( 'default' );
+
+		parent::tear_down();
+	}
+
 	/**
 	 * @ticket 23311
 	 */
 	public function test_wp_phpmailer_error_message_keys_match() {
-		require_once ABSPATH . WPINC . '/PHPMailer/PHPMailer.php';
-		require_once ABSPATH . WPINC . '/class-wp-phpmailer.php';
 
 		$phpmailer = new PHPMailer\PHPMailer\PHPMailer();
 		$phpmailer->SetLanguage();
 
-		$wp_phpmailer = new WP_PHPMailer();
+		$wp_phpmailer = new MockPHPMailer();
+
+		$this->assertEqualSets( array_keys( $phpmailer->GetTranslations() ), array_keys( $wp_phpmailer->GetTranslations() ) );
+	}
+
+	/**
+	 * @ticket 23311
+	 */
+	public function test_phpmailer_error_messages_translation() {
```

Comment:
Be more specific, what the function does. It does not make sense to just add ticket number.

Provide some good function doc and then add ticket link insted of number.

Reply:
Hi @hbhalodia, Ticket numbers help to test those specific tickets, also in other test files, they are using ticket numbers.

Reply:
I agree, You can add the ticket link instead of number to quickly go through tickets. Also, I say in addition with this, please add some function doc as well.

Something like this,

https://github.com/WordPress/wordpress-develop/blob/32880ec4e910837cd267e1685733867b243e22fa/tests/phpunit/tests/comment/commentTime.php#L47-L56